src/uct/ib/base/ib_md.h

```diff
@@ -166,6 +166,7 @@ typedef struct uct_ib_md_config {
     unsigned                 devx_objs;    /**< Objects to be created by DevX */
     ucs_on_off_auto_value_t  mr_relaxed_order; /**< Allow reorder memory accesses */
     int                      enable_gpudirect_rdma; /**< Enable GPUDirect RDMA */
+    int                      force_rdma_transport; /**< Force the RDMA transport enabling*/
```

Comment:
1. seems unused
2. maybe rename to "ignore_memlock_limit" ?

Reply:
it was decided that this feature may be merged without user side control. So this field should be removed.